lib/workos/user_management.rb

```diff
@@ -462,6 +462,40 @@ def authenticate_user_with_code(code:, client_id:, ip_address: nil, user_agent:
         )
         WorkOS::UserResponse.new(response.body)
       end
+      #
+      # Authenticates a user using TOTP.
+      #
+      # @param [String] code The one-time code that was emailed to the user.
+      # @param [String] client_id The WorkOS client ID for the environment
+      # @param [String] ip_address The IP address of the request from the user who is attempting to authenticate.
+      # @param [String] user_agent The user agent of the request from the user who is attempting to authenticate.
+      #
+      # @return WorkOS::UserResponse
+
+      sig do
+        params(
+          code: String,
+          client_id: String,
+          pending_authentication_token: String,
+          authentication_challenge_id: String,
```

Comment:
Looks like we are missing `ip_address` and `user_agent` as optional parameters:

```suggestion
          code: String,
          client_id: String,
          pending_authentication_token: String,
          authentication_challenge_id: String,
          ip_address: T.nilable(String),
          user_agent: T.nilable(String),
```

Reply:
Derp, actually this is just bad doc copy pasted on my part. Will remove these from the description docs